app/models/ama.rb

```diff
@@ -0,0 +1,62 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: amas
+#
+#  id                    :integer          not null, primary key
+#  ama_subscribers_count :integer          default(0), not null
+#  description           :string           not null
+#  end_date              :datetime         not null
+#  start_date            :datetime         not null
```

Comment:
I dont know, @JoshFabian: do we want to create AMAs that dont have dates or original posts?